openmc/mgxs/mgxs.py

```diff
@@ -425,7 +427,11 @@ def get_mgxs(mgxs_type, domain=None, domain_type=None,
 
         Parameters
         ----------
-        mgxs_type : {'total', 'transport', 'nu-transport', 'absorption', 'capture', 'fission', 'nu-fission', 'kappa-fission', 'scatter', 'nu-scatter', 'scatter matrix', 'nu-scatter matrix', 'chi'}
+        mgxs_type : {'total', 'transport', 'nu-transport', 'absorption',
```

Comment:
This was a single long line in order for each type to be properly tracked by Sphinx.